USH_Apps/NY_CICT/Utilities/user_inputs.robot

```diff
@@ -9,4 +9,4 @@ ${pass}    %{DIMAGIQA_RF_PASSWORD}
 ${implcit_wait_time}    20
 ${app name}    NY
 ${archieved_contact_name}     Dewey
-${archieved_contact_lname}      SEPTEMBER13
\ No newline at end of file
+${archieved_contact_lname}      SEPTEMBER13
```

Comment:
Will these value remain the same everytime? Will the testcases pass?

Reply:
I've skipped these test cases, like we discussed last time. One has to create archived contacts manually and update the value here every time.